snuba/admin/auth_roles.py

```diff
@@ -173,6 +182,10 @@ def generate_tool_test_role(tool: str) -> Role:
         name="AllMigrationsExecutor",
         actions={ExecuteAllAction(list(MIGRATIONS_RESOURCES.values()))},
     ),
+    "ClickhouseAdmin": Role(
+        name="clickhouse-admin",
+        actions={ExecuteSudoSystemQuery([TOOL_RESOURCES["sudo-system-queries"]])},
+    ),
```

Comment:
This role does not need specify resources as of today. It could be extended in order to specify which clusters to use. Should I just remove the ROLL_RESOURCE ?